aten/src/ATen/native/native_functions.yaml

```diff
@@ -4988,6 +4988,8 @@
 
 - func: polygamma_(Tensor(a!) self, int n) -> Tensor(a!)
   variants: method
+  dispatch:
+    CompositeExplicitAutograd: polygamma_
```

Comment:
Implementation is

```
Tensor& polygamma_(Tensor& self, int64_t n) {
  return at::polygamma_out(self, n, self);
}
```

so this looks overly restrictive

Reply:
Sorry didn't get you.

Reply:
CompositeImplicitAutograd is probably more correct, since *if* `polygamma_out` had supported AD, then `polygamma_` would too. (It's not too important though, because it doesn't support AD lol)